crane/app_util.py

```diff
@@ -149,6 +149,33 @@ def get_data():
     return request.crane_data
 
 
+def get_repositories():
+    """
+    Get the current data used for processing requests from the flask request context
+    and format it to display basic information about image ids and tags associated
+    with each repository.
+
+    Value corresponding to each key(repo-registry-id) is a dictionary itself
+    with the following format:
+    {'image-ids': [<image-id1>, <image-id2>, ...],
+     'tags': {<tag-id1>: <tag1>, <tag-id2>: <tag2>, ...}
+     'protected': true/false}
+
+    :return: dictionary keyed by repo-registry-ids
+    :rtype: dict
+    """
+    all_repo_data = get_data()['repos']
```

Comment:
I think this will cause a key error if the json file is inavlid or missing 'repos'. Maybe have this use .get() with a default so if repos is missing as a key it shows empty?